charts/foreman/values.yaml

```diff
@@ -1,7 +1,7 @@
 ## @section Global configuration options
 ##
 global:
-  ## @param nameOverride String to partially override common.names.fullname
```

Comment:
Could you make this change part of a separate PR (and bump the Chart version to avoid CI failure)?